### PR TITLE
Update the description of HDS on website to clarify its intended scope

### PR DESCRIPTION
**Pull request by didoo**
### :pushpin: Summary

This PR updated the `About` and `Overview` pages of the website, to clarify what is the (main) scope of the design system, to make sure it's clear it's not specifically intended for internal products/tools (there may be exceptions, but they need to be discussed/agreed with the HDS team).

### :link: External links

Jira ticket: https://hashicorp.atlassian.net/browse/HDS-3210

***

### 👀 Component checklist

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

File: website/docs/about/overview.md [hunk 1]
```diff
@@ -6,45 +6,63 @@ navigation:
...
+
+Helios is **not** meant to be used as a general-purpose UI library.
+
+If you intend to use Helios for **internal products or tools** in HashiCorp, please [contact the Design Systems Team](/about/support) before starting so we can discuss if Helios is a good fit for your team and how this might affect system overhead (changes, maintenance, support, etc).
+
+### External products
+
+Helios is **not** meant to be used outside of HashiCorp. 
+
+If you’re **not a HashiCorp employee** you’re more than welcome to use the Helios design system as reference, but we don’t provide support if using it to build your own product or UIs. At any moment we may introduce changes to our assets or components that could potentially break your product or UIs.
```
Comment (didoo) [inline website/docs/about/overview.md:43]:
```suggestion
If you’re **not a HashiCorp employee** you’re more than welcome to use the Helios design system as reference, but we don’t provide support if you're using it to build your own product or UIs. At any moment we may introduce changes to our assets or components that could potentially break your product or UIs.
```

Comment (heatherlarsen) [reply website/docs/about/overview.md:43]: Grammarly suggestions: 
```suggestion
If you’re **not a HashiCorp employee**, you’re more than welcome to use the Helios design system as a reference, but we don’t provide support if you use it to build your product or UIs. At any moment, we may introduce changes to our assets or components that could potentially break your product or UIs.
```

File: website/docs/about/overview.md [hunk 2]
```diff
@@ -6,45 +6,63 @@ navigation:
...
 Design systems are often equated with a UI kit or component library. In reality, an effective design system spans the entire lifecycle of a product, from ideation to iteration and everything in between.
 
-## Helios architecture
+## What is Helios
 
-While the scale of a design system largely depends on the needs of those consuming it, the fundamental structure and architecture remain largely the same across systems.
+**Helios** is the design system developed by the HashiCorp Design System Team. It’s commonly referred to as **HDS**.
 
-In Helios, we offer the following:
+Helios is available and recommended for all HashiCorp **product teams using Ember**. We also support teams using alternative frameworks or libraries, such as HashiCorp Developer, via our design assets, [tokens](/foundations/tokens), and best practice guidance.
```
Comment (Dhaulagiri) [inline website/docs/about/overview.md:19]:
```suggestion
Helios is available and recommended for all HashiCorp **product teams using Ember.js**. We also support teams using alternative frameworks or libraries, such as HashiCorp Developer, via our design assets, [tokens](/foundations/tokens), and best practice guidance.
```

File: website/docs/about/overview.md [hunk 3]
```diff
@@ -6,45 +6,63 @@ navigation:
...
+Using Helios ensures the different product lines share a common design language. We work closely with these teams on integration and implementation while collecting feedback to understand consumer needs and inform our [roadmap](https://go.hashi.co/hds-rollout).
 
-We structure our components, icons, and design tokens in a [monorepo](https://github.com/hashicorp/design-system) using yarn workspaces and publish a collection of libraries in [Figma](https://www.figma.com/files/team/1030156573400567478).
+For more information on how to use the design system, visit our getting started guides for [designers](/getting-started/for-designers) and for [engineers](/getting-started/for-engineers) or visit our public [GitHub](https://github.com/hashicorp/design-system) repository.
+
+## When to use Helios
+
+### Consumer-facing products
+
+Helios was specifically created for **consuming-facing products**.
```
Comment (Dhaulagiri) [inline website/docs/about/overview.md:29]:
```suggestion
Helios was specifically created for **customer-facing products**.
```

File: website/app/templates/about.hbs [hunk 4]
```diff
@@ -10,6 +10,8 @@
   <Doc::Page::Content>
     <p class="doc-text-body">Helios is a flexible, scalable design system built by HashiCorp on the foundations of
       accessibility, beautiful design, and consistent user experience.</p>
+    <p class="doc-text-body">It is used by the HashiCorp product teams to design and build the UIs
+      of their <strong>consumer-facing products</strong>.</p>
```
Comment (Dhaulagiri) [inline website/app/templates/about.hbs:14]:
```suggestion
      of their <strong>customer-facing products</strong>.</p>
```

File: website/docs/about/overview.md [hunk 5]
```diff
@@ -6,45 +6,63 @@ navigation:
...
-- **Patterns**: repetitive combinations of components and foundational styles that yield more complex functions and features
-- **Documentation**: education, issue tracking, contribution, content management, processes
+Using Helios ensures the different product lines share a common design language. We work closely with these teams on integration and implementation while collecting feedback to understand consumer needs and inform our [roadmap](https://go.hashi.co/hds-rollout).
 
-We structure our components, icons, and design tokens in a [monorepo](https://github.com/hashicorp/design-system) using yarn workspaces and publish a collection of libraries in [Figma](https://www.figma.com/files/team/1030156573400567478).
+For more information on how to use the design system, visit our getting started guides for [designers](/getting-started/for-designers) and for [engineers](/getting-started/for-engineers) or visit our public [GitHub](https://github.com/hashicorp/design-system) repository.
+
+## When to use Helios
+
+### Consumer-facing products
```
Comment (Dhaulagiri) [inline website/docs/about/overview.md:27]:
```suggestion
### Customer-facing products
```

File: website/docs/about/overview.md [hunk 6]
```diff
@@ -6,45 +6,63 @@ navigation:
...
 
 A design system is a set of guidelines, standards, assets, and processes to help organizations design, build, and deploy products rapidly and consistently.
 
 Design systems are often equated with a UI kit or component library. In reality, an effective design system spans the entire lifecycle of a product, from ideation to iteration and everything in between.
 
-## Helios architecture
+## What is Helios
 
-While the scale of a design system largely depends on the needs of those consuming it, the fundamental structure and architecture remain largely the same across systems.
+**Helios** is the design system developed by the HashiCorp Design System Team. It’s commonly referred to as **HDS**.
```
Comment (jorytindall) [inline website/docs/about/overview.md:17]: Not opinionated on this (and there's obviously other context on this page), but using the term 'developed' kind of reads to me like a engineering-focused effort. Is something agnostic better? Built? Created?
```suggestion
**Helios** is the design system built by the HashiCorp Design System Team. It’s commonly referred to as **HDS**.
```

**Review by jorytindall — APPROVED** [review]
Just left a small (very) nitpicky comment. Feel free to disregard if it's not relevant! Otherwise this looks good!

File: website/docs/about/overview.md (hunk 6 above)
Comment (heatherlarsen) [reply website/docs/about/overview.md:17]: Ah, that's a good catch. I like this adjustment.

File: website/app/templates/about.hbs [hunk 7]
```diff
@@ -10,6 +10,8 @@
   <Doc::Page::Content>
     <p class="doc-text-body">Helios is a flexible, scalable design system built by HashiCorp on the foundations of
       accessibility, beautiful design, and consistent user experience.</p>
+    <p class="doc-text-body">It is used by the HashiCorp product teams to design and build the UIs of their
```
Comment (KristinLBradley) [inline website/app/templates/about.hbs:13]:
```suggestion
    <p class="doc-text-body">It is used by HashiCorp product teams to design and build the UIs of their
```

File: website/docs/about/overview.md [hunk 8]
```diff
@@ -6,45 +6,63 @@ navigation:
...
 Design systems are often equated with a UI kit or component library. In reality, an effective design system spans the entire lifecycle of a product, from ideation to iteration and everything in between.
 
-## Helios architecture
+## What is Helios
 
-While the scale of a design system largely depends on the needs of those consuming it, the fundamental structure and architecture remain largely the same across systems.
+**Helios** is the design system built by the HashiCorp Design System Team. It’s commonly referred to as **HDS**.
 
-In Helios, we offer the following:
+Helios is available and recommended for all HashiCorp **product teams using Ember.js**. We also support teams using alternative frameworks or libraries, such as HashiCorp Developer, via our design assets, [tokens](/foundations/tokens), and best practice guidance.
```
Comment (KristinLBradley) [inline website/docs/about/overview.md:19]: The phrasing of this part:

_"We also support teams using alternative frameworks or libraries, **such as HashiCorp Developer**"_

to me kind of sounds like "HashiCorp Developer" itself is an alternative framework/library...

**How about changing to:**
"We also support teams using alternative frameworks or libraries, such as React, via our design assets, [tokens](/foundations/tokens), and best practice guidance."

-OR-

"We also support teams using alternative frameworks or libraries, such as the HashiCorp Developer team which uses React, via our design assets, [tokens](/foundations/tokens), and best practice guidance."

**Review by KristinLBradley — APPROVED** [review]
Looks good. Just added a couple of suggested edits.

File: website/docs/about/overview.md (hunk 8 above)
Comment (Dhaulagiri) [inline website/docs/about/overview.md:19]:
```suggestion
Helios is available and recommended for all HashiCorp product teams using Ember.js. We also support teams using alternative frameworks or libraries, such as React, on a case-by-case basis via our design assets, [tokens](/foundations/tokens), and best practice guidance.
```

Comment (KristinLBradley) [reply website/docs/about/overview.md:19]: See the related suggested edit @Dhaulagiri has now added: https://github.com/hashicorp/design-system/pull/2012#discussion_r1544481682

Comment (KristinLBradley) [reply website/docs/about/overview.md:19]: Nice

**Pull request merged**